Document/0x06b-Basic-Security-Testing.md

```diff
@@ -96,54 +96,47 @@ The iOS jailbreak scene evolves so rapidly that providing up-to-date instruction
 
-```http
-http://repo.hackyouriphone.org
-```
+You can also easily install Frida by adding the following repository to Cydia <https://build.frida.re>.
 
 The following are some useful packages you can install from Cydia to get started:
 
 - BigBoss Recommended Tools: Installs many useful command line tools for security testing including standard Unix utilities that are missing from iOS, including wget, unrar, less, and sqlite3 client.
 - adv-cmds: Advanced command line. Includes finger, fingerd, last, lsvfs, md, and ps.
 - [IPA Installer Console](https://cydia.saurik.com/package/com.autopear.installipa/ "IPA Installer Console"): Tool for installing IPA application packages from the command line. Package name is `com.autopear.installipa`.
 - Class Dump: A command line tool for examining the Objective-C runtime information stored in Mach-O files.
-- Cydia or Mobile Substrate: A platform that makes developing third-party iOS add-ons easier via dynamic app manipulation or introspection.
-- cycript: Cycript is an inlining, optimizing, Cycript-to-JavaScript compiler and immediate-mode console environment that can be injected into running processes (assiciated to Substrate).
+- Mobile Substrate: A platform that makes developing third-party iOS add-ons easier via dynamic app manipulation or introspection.
+- Cycript: Is an inlining, optimizing, Cycript-to-JavaScript compiler and immediate-mode console environment that can be injected into running processes (associated to Substrate).
 - AppList: Allows developers to query the list of installed apps and provides a preference pane based on the list.
 - PreferenceLoader: A Mobile Substrate-based utility that allows developers to add entries to the Settings application, similar to the SettingsBundles that App Store apps use.
 - AppSync Unified: Allows you to sync and install unsigned iOS applications.
 
-#### Recommended Tools
+Besides Cydia there are several other open source tools available and should be installed, such as [Introspy](https://github.com/iSECPartners/Introspy-iOS "Introspy-iOS").
+
+#### Recommended Tools - macOS Device
 
 In order to analyse iOS apps, you should use a macOS device and install the following tools we'll be referring throughout the guide:
 
-- [Introspy](https://github.com/iSECPartners/Introspy-iOS "Introspy-iOS")
+- [Burp Suite](https://portswigger.net/burp "Burp Suite")
 - [Frida](https://www.frida.re "Frida")
 - [IDB](https://www.idbtool.com "IDBTool")
-- [Burp Suite](https://portswigger.net/burp "Burp Suite")
 - [Needle](https://github.com/mwrlabs/needle "Needle"): Is an all-in-one iOS security assessment framework. The [installation guide](https://github.com/mwrlabs/needle/wiki/Installation-Guide "Needle Installation Guide") in the Github wiki contains all the information needed on how to prepare your Kali Linux or macOS and how to install the Needle Agent on your iOS device.
```

Comment:
ios-deploy, Objection, MobSF, Frida-ios-dump, iFunbox, https://github.com/mechanico/Keychain-Dumper, Ghidra, Radare2, and TablePlus

Reply:
done